rbd/rbd.go

```diff
@@ -225,20 +225,23 @@ func GetImageNames(ioctx *rados.IOContext) (names []string, err error) {
 	}
 }
 
-//
+// GetImage gets a reference to a previously created rbd image.
 func GetImage(ioctx *rados.IOContext, name string) *Image {
 	return &Image{
 		ioctx: ioctx,
 		name:  name,
 	}
 }
 
-// int rbd_create(rados_ioctx_t io, const char *name, uint64_t size, int *order);
+// Create a new rbd image.
 //
-// But also (for backward compability):
-// int rbd_create2(rados_ioctx_t io, const char *name, uint64_t size,
+// Implements:
+//  int rbd_create(rados_ioctx_t io, const char *name, uint64_t size, int *order);
+//
+// Also implements (for backward compatibility):
+//  int rbd_create2(rados_ioctx_t io, const char *name, uint64_t size,
 //          uint64_t features, int *order);
-// int rbd_create3(rados_ioctx_t io, const char *name, uint64_t size,
+//  int rbd_create3(rados_ioctx_t io, const char *name, uint64_t size,
```

Comment:
Have you considered indenting the follow-up lines with the arguments of the C API function somewhat? It is currently a little uneven.

Reply:
I thought about it but wanted to keep the changes more minimal for now.